src/Controls/samples/Controls.Sample/Controls.Sample.csproj

```diff
@@ -3,12 +3,6 @@
     <TargetFrameworks>netstandard2.1;Xamarin.iOS10;MonoAndroid10.0</TargetFrameworks>
   </PropertyGroup>  
   <Import Project="..\..\..\..\.nuspec\Microsoft.Maui.Controls.MultiTargeting.targets" />
-  <!--<ItemGroup Condition="$(TargetFramework.StartsWith('Xamarin.iOS')) == true ">
```

Comment:
these projects don't compile with msbuild for some reason. I don't think sdk.extras can handle the conditionals